### PR TITLE
Added documentation for the order endpoint

### LINKED ISSUE #332: Missing documentation for OrderList endpoint
https://github.com/Sylius/ShopApiPlugin/pull/314#issuecomment-431076952

### PR DESCRIPTION
**Pull request by mamazu**
Closes #332

### CODE REVIEW

**Comment by mamazu** [comment]
Could you review this as well? @lchrusciel

**Comment by lchrusciel** [comment]
I have done it already but forgot to submit 🤦‍♂️

**Comment by mamazu** [comment]
Ah, okay then I will close it here. :D

**Comment by lchrusciel** [comment]
NOOOOOO

**Comment by lchrusciel** [comment]
I meant review only. PR is 100% valid, can you just add this short fix?

**Comment by mamazu** [comment]
Ah, sure.

File: doc/swagger.yml [hunk 1]
```diff
@@ -883,6 +888,21 @@ paths:
...
+            - "order"
+            summary: "Shows a list of orders of the customer"
+            operationId: "showsOrderId"
+            responses:
+                200:
+                    description: "Shows a list of placed orders of the customer"
+                    type: "array"
+                    items:
+                        type: "object"
+                        $ref: "#/definitions/Cart"
```
[the remark this answers is not in the document]
Comment (mamazu) [reply doc/swagger.yml]: Of course

**Comment by lchrusciel** [comment]
Thanks Maximillian! 🐋

And sorry for confusion :D

**Comment by mamazu** [comment]
Sure no problem.

**Pull request merged**